Common/ArmoniK.DevelopmentKit.Common/Submitter/BaseClientSubmitter.cs

```diff
@@ -499,9 +520,15 @@ public byte[] TryGetResult(string taskId, bool checkOutput = true, CancellationT
                                                  case Output.TypeOneofCase.Ok:
                                                    break;
                                                  case Output.TypeOneofCase.Error when !string.IsNullOrEmpty(taskOutput.Error.Details):
-                                                   throw new Exception($"Task in Error - {taskId} : {taskOutput.Error.Details}");
+                                                   throw new ClientResultsException($"Task in Error - {taskId} : {taskOutput.Error.Details}",
+                                                                                    taskId);
+                                                 default:
+                                                   throw new ArgumentOutOfRangeException();
                                                }
 
+                                               if (response == null)
+                                                 throw new ClientResultsException($"Unknown error to retrieve result of task {taskId}",
```

Comment:
[format] Curly brackets around
[english] "Unknown error while retrieving result of task" ?

Reply:
Done